internal/constants.go

```diff
@@ -13,6 +14,8 @@ const (
 	MainCanvasWidth                = 800
 )
 
+var SupportedGPTModels = []string{openai.GPT3Dot5Turbo, openai.GPT4, openai.GPT4TurboPreview}
```

Comment:
Why limit to this list?

Reply:
no reason, wasnt sure how much to add so left the basics here.

Reply:
I think we should keep a complete list here.

Reply:
added all but soon to be deprecated ones (Jan 4th 2024)